JobConfig/pileup/MuStopPileup.fcl

```diff
@@ -10,7 +10,6 @@
 #include "Production/JobConfig/primary/prolog.fcl"
 #include "Production/JobConfig/pileup/prolog.fcl"
 #include "fcl/standardServices.fcl"
-#include "EventGenerator/fcl/prolog.fcl"
```

Comment:
Did this file cause conflicts?  It shouldn't be necessary so removing it is fine.  It is full of obsolete declarations.